File: 2st/Riyeon/옹알이 (1).py
def solution(babbling):
    answer = 0
    li = ['aya', 'ye', 'woo', 'ma']
    for b in babbling:
        for l in li:
            b = b.replace(l, '*')
        b = b.replace('*', '')
        if len(b) == 0:
            answer += 1
    return answer

File: 3st/Juhee/옹알이 (1).java
class Solution {
    public int solution(String[] babbling) {
        String[] str = {"aya", "ye", "woo", "ma"};
        int answer = 0;
        
        for(int i=0; i<babbling.length; i++){
            for(int j=0; j<str.length; j++){
                babbling[i] = babbling[i].replace(str[j], " ");
            }
            
            if(babbling[i].trim().length() == 0){
                answer++;
            }
        }
        return answer;
    }
}
